code/index.d.ts

```diff
@@ -1,26 +1,28 @@
-// Type definitions for code 4.0.0
+// Type definitions for code 4.0
 // Project: https://github.com/hapijs/code
 // Definitions by: Prashant Tiwari <https://github.com/prashaantt>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /** Generates an assertion object. */
-export function expect<T>(value: T | T[], prefix?: string): AssertionChain<T>;
+declare function expect<T>(value: T | T[], prefix?: string): AssertionChain<T>;
```

Comment:
I'm not sure, this particular lib's API is entirely expressed as chains: `expect(1).to.not.equal(2)` etc. Once you get hold of a top level function. you can work your way through any level of depths. I don't know when one might need to rely on the interfaces.

I understand this is breaking change for anybody who may have come to rely on them, but this was one of the very first full type defs I submitted and always wondered how to fix this issue. I've seen other definitions since then and for this PR I took inspiration from one of them.

I'll defer to your final call in case there's a well-defined best practice around this which I'm not aware of, but for now this just feels right to me.